Tasks/VSBuildV1/VSBuild.ps1

```diff
@@ -49,6 +49,7 @@ try {
     $msBuildVersion = $null;
     switch ("$vsVersion") {
         '' { $msBuildVersion = '14.0' ; break } # VS wasn't found. Attempt to find MSBuild 14.0 or lower.
+        '16.0' { $msBuildVersion = '16.0' ; break }
```

Comment:
make sure to bump the version for any tasks that consume the msbuildhelpers module